### ISSUE TITLE
Unable to see graphs when multiple profile results are available

### ISSUE THREAD

**Issue by dcamper** [hpcc-systems/DataPatterns#45]
If you run an ECL job where there are multiple data profiling results, no result graphs show up in the Resources tab (though there may be a lot of hidden content, based on how the scroll bar looks).

**Comment by dcamper** [comment]
@jbrundage Can you take a look at this?

**Comment by jbrundage** [comment]
@dcamper Sure I'll investigate. Are there any examples on play.hpccsystems.com where this issue can be seen? If not, how can an ECL job have multiple data profiling results?

**Comment by dcamper** [comment]
@jbrundage Here is a link to a job I just ran:

http://play.hpccsystems.com:8010/?Wuid=W20190610-114953&Widget=WUDetailsWidget#/stub/Summary

Interestingly, this shows a different behavior than what I reported.  This job shows the result from the first profile and ignores the second.  What I saw originally was an absence of any output.  I don't recall what version of the platform my original job ran on, but that may have had something to do with it.

At any rate, it would be great if all profiles could be shown, perhaps via a drop-down menu.
